addons/website_sale/models/website.py

```diff
@@ -15,6 +15,7 @@
 FISCAL_POSITION_SESSION_CACHE_KEY = 'fiscal_position_id'
 PRICELIST_SESSION_CACHE_KEY = 'website_sale_current_pl'
 PRICELIST_SELECTED_SESSION_CACHE_KEY = 'website_sale_selected_pl_id'
+SHOP_PATH = '/shop'
```

Comment:
put it in a const.py then (no need to migrate the other ones, I'll do it myself later on).